tests/filetypes/TestOpenPackagingConvention.py

```diff
@@ -89,6 +91,29 @@ def test_cycleSetDataGetData(virtual_path: str):
     assert result[virtual_path] == test_data #The data itself is still correct.
 
 
+@pytest.mark.parametrize("virtual_path, path_list", [
+    ("/foo/materials", ["/foo/materials", "/[Content_Types].xml", "/_rels/.rels"]),
+    ("/materials", ["/files/materials", "/[Content_Types].xml", "/_rels/.rels"])
+])
+def test_aliases_replacement(virtual_path: str, path_list: List[str]):
+    test_data = b"Let's see if we can read this data back."
+
+    stream = io.BytesIO()
```

Comment:
Suggestion: maybe use a context manager?

Reply:
The reason why to use stream here is to call the stream handling method directly. That method actually triggers the alias handling method (this is what we want to test). Furthermore, this is how other tests in the suite were set up, just keeping the same with others IMO is fine - seeing a context manager text, and others doing something different, seems a bit strange 🤔